spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/jms/DefaultJmsListenerContainerFactoryConfigurer.java

```diff
@@ -89,50 +49,26 @@ void setTransactionManager(JtaTransactionManager transactionManager) {
 		this.transactionManager = transactionManager;
 	}
 
-	/**
-	 * Set the {@link JmsProperties} to use.
-	 * @param jmsProperties the {@link JmsProperties}
-	 */
-	void setJmsProperties(JmsProperties jmsProperties) {
-		this.jmsProperties = jmsProperties;
-	}
-
 	/**
 	 * Set the {@link ObservationRegistry} to use.
 	 * @param observationRegistry the {@link ObservationRegistry}
 	 * @since 3.2.1
 	 */
+	@Override
 	public void setObservationRegistry(ObservationRegistry observationRegistry) {
-		this.observationRegistry = observationRegistry;
+		super.setObservationRegistry(observationRegistry);
 	}
```

Comment:
This override is here to preserve backwards compatibility (see #44466), as `AbstractJmsListenerContainerFactoryConfigurer` introduced in this PR reduces visibility to package private.